myproject/blog/views.py

```diff
@@ -40,7 +43,7 @@ class BlogDetail(generic.DetailView,FormMixin):
     def get_context_data(self, **kwargs: Any):
         context = super().get_context_data(**kwargs)
         post = self.get_object()
-        context['comments'] = post.comments.all().order_by("created_at")
+        context['comments'] = Comment.objects.select_related().filter(blog = post)
```

Comment:
Explain me how this will work. Also let me know how you will check if this has improved the query performance or not.